reference-architecture/vmware-ansible/vars/infrastructure.yaml

```diff
@@ -0,0 +1,83 @@
+---
+infrastructure_hosts:
```

Comment:
VMs you can annotate. Hosts you can't. Tagging in VMware is a PITA atm. There is a new rest API in the works thats supposed to help with this.